resources/build/test/test.sh

```diff
@@ -184,6 +192,14 @@ fi
 echo -e "${COLOR_BLUE}## Testing output of: builder_parse --feature xyzzy --bar abc --baz def test${COLOR_RESET}"
 parse_output=$(builder_parse --feature xyzzy --bar abc --baz def test)
 expected="$(builder_echo setmark "test.sh parameters: <--feature xyzzy --bar abc --baz def test>")"
+
+if [[ $VERSION_ENVIRONMENT == "local" ]]; then
+  expected="$(builder_echo setmark "test.sh parameters: <--feature xyzzy --bar abc --baz def test --debug>")"
+  # Yay, the fun of getting the second line... that newline before the closing brace is intentional.  :(
```

Comment:
The "Local build environment detected" note for local builds will appear in the captured output when it occurs!